waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -508,28 +513,33 @@ proc getNumStreams*(pm: PeerManager, protocol: string): (int, int) =
               numStreamsOut += 1
   return (numStreamsIn, numStreamsOut)
 
+proc pruneInRelayConns(pm: PeerManager, amount: int) {.async.} =
+  let (inRelayPeers, outRelayPeers) = pm.connectedPeers(WakuRelayCodec)
+  let connsToPrune = min(amount, inRelayPeers.len)
+
+  for p in inRelayPeers[0..<connsToPrune]:
+    await pm.switch.disconnect(p)
+
 proc connectToRelayPeers*(pm: PeerManager) {.async.} =
```

Comment:
I'd rename this proc to something like `updateRelayPeersConnections`

Reply:
wops i missed this. isn't `connectToRelayPeers` shorter and more descriptive?

Reply:
Hey @alrevuelta ! It is shorter but this proc not only `connectToNodes` but also `pruneInRelayConns`. This is why I see it more convenient to have the "update" word on its name. Maybe sth like `updateRelayPeers` can suit better, which is even shorter ;P